pkg/session/session.go

```diff
@@ -114,36 +116,16 @@ func GetCurrentSession() *session {
 
 var currentSession *session
 
-func createDefaultSession() error {
-	file, err := os.Create(constants.OpenKommanderConfigFilename)
-	if err != nil {
-		fmt.Println("Error creating session file:", err)
-		return err
-	}
-	defer func() {
-		if err := file.Close(); err != nil {
-			fmt.Printf("Error closing file: %v\n", err)
-		}
-	}()
-
-	sessionData := SessionData{Brokers: []string{}, IsAuthenticated: false, Version: constants.SaramaKafkaVersion.String()}
-	return json.NewEncoder(file).Encode(sessionData)
-}
-
 func saveSession() error {
-	file, err := os.Create(constants.OpenKommanderConfigFilename)
-	if err != nil {
-		fmt.Println("Error creating session file:", err)
-		return err
-	}
-	defer func() {
-		if err := file.Close(); err != nil {
-			fmt.Printf("Error closing file: %v\n", err)
-		}
-	}()
-
-	sessionData := SessionData{Brokers: currentSession.brokers, IsAuthenticated: currentSession.isAuthenticated, Version: currentSession.version.String()}
-	err = json.NewEncoder(file).Encode(sessionData)
+	envMap := make(map[string]string)
```

Comment:
I'm a bit confused, did we move the config file to the env file? I am not a big fan of introducing a third party library for a trivial task like writing to a file, so if we decide this is the direction we want to take, we can easily implement this functionality ourselves.

Or we can have a hybrid solution where the broker metadata is stored in a flat file on disk and only the credentials are preserved in the env variable. 

@DuarteMartinho what do you think?